pkg/applyinator/applyinator.go

```diff
@@ -28,6 +29,38 @@ type Applyinator struct {
 	imageUtil       *image.Utility
 }
 
+// CalculatedPlan is passed into Applyinator and is a Plan with checksum calculated
+type CalculatedPlan struct {
+	Plan     Plan
```

Comment:
Will this value be mutated by the applyinator?

Reply:
This value generally is not mutated by the applyinator